src/test/Fake.Core.UnitTests/Fake.DotNet.Testing.SpecFlow.fs

```diff
@@ -0,0 +1,29 @@
+module Fake.DotNet.Testing.SpecFlowTests
+
+open System.IO
+open Fake.Core
+open Fake.DotNet
+open Fake.DotNet.Testing
+open Fake.Testing
+open Expecto
+
+[<Tests>]
+let tests =
+  testList "Fake.DotNet.Testing.SpecFlow.Tests" [
+    testCase "Test that new argument generation works" <| fun _ ->
```

Comment:
@magicmonty Can you take a look at this test (if it is reasonably) and maybe add some to the other PRs as well?

Reply:
Do we need to run the specflow exe with mono when not on windows?

Reply:
@matthid 

> Do we need to run the specflow exe with mono when not on windows?
I don't know. I suppose so.

Reply:
> @magicmonty Can you take a look at this test (if it is reasonably) and maybe add some to the other PRs as well?

Sure, I have a look